readme.md

```diff
@@ -1,168 +1,161 @@
+  - In Sinatra, our routes and controller code live in the same file. This will change with Rails.
+4. In most cases, the controller will use ActiveRecord methods to...
+  - Create, read, update or destroy data in the **database**.
+5. Once the **controller** has done its job and gotten any data it needs, it renders a **view** or redirects to a new **request**, which is displayed back to the user in the browser.
 
 ![Sinatra MVC](images/sinatra_mvc2.jpg)
 
+<!-- AM: Along with diagram, write out quick numbered list indicating order. -->
 
 ## App Organization
 
-Our apps are starting to have lots of different dependencies. It's very important to start separating our concerns.
+As our apps get more complex, they begin to require lots of different dependencies. It's very important to start separating our concerns.
+
+Here's a summary of the important folders and files in our app. It's important to note that some of these are expected by sinatra (e.g., `views`) and the rest are just convention to match our upcoming transition to Rails.
+
+**`tunr`**
+- **`db`**
+  - Seeds (sample data)
+  - Database configuration and connection files
+- **`models`**
+  - Classes that inherit from ActiveRecord
+- **`controllers`**
+  - Route definitions (e.g., `get "/artists" do`)
+  - `app.rb`
+- **`views`**
+  - `.erb` templates
+- **`public`**
+  - Static assets (e.g., JS, CSS, images, fonts)
+- *app.rb*
+  - The entry point to our application
+  - Start the application with `ruby app.rb`
+- *console.rb*
+  - Interact with models and data in the database via REPL.
 
-Here's a summary of the important folders / files in our app. It's important to note that some of these are expected by sinatra (`views`), and the rest are just convention to match Rails.
+For this lesson, we'll be using [Tunr](http://github.com/ga-dc/tunr_sinatra). Go ahead and clone it and change directories into the `tunr` folder.
 
-* **tunr**
-  - **db**
-    - seeds (sample data)
-    - database configuration and connection files
-  - **models**
-    - classes that inherit from ActiveRecord
-  - **controllers**
-    - route definitions (`get "/artists" do`)
-    - for now `app.rb`
-  - **views**
-    - erb templates
-  - **public**
-    - static assets
-      - JS, CSS, images, fonts
-  - *app.rb*
-    - the entry point to our application
-    - start the application with `ruby app.rb`
-  - *console.rb*
-    - interact with models and data in the database
+### Goal: Seed the database.
 
-For this lesson, we'll be using [Tunr](http://github.com/ga-dc/tunr_sinatra), so go ahead and clone it and change directories into the `tunr` folder.
+In this repo are three files of note: `artist_data.rb`, `song_data.rb` and `seeds.rb`. We want to use the `seeds.rb` file to seed our database.
 
-### Goal - Seed the database.
+#### Turn & Talk (10 minutes / 0:25)
 
```

Comment:
How high value is this T & T? Feel like if you needed to cut something it could be this...